gigamon/logs/Gigamon.yml

```diff
@@ -0,0 +1,128 @@
+id: gigamon
+# See app_id in your integration's manifest.json file to learn more:
+# https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
+metric_id: gigamon
+# If for some reason id must be different than app_id, add the app_id in this field instead.
+# If id and app_id already match, this field can be left blank.
+facets:
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_name
+    path: dns_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - others
```

Comment:
We don't want integrations to have multiple facet groups. There should be only one custom group which you have for `gigamon`. Any other facets that aren't custom to your integration, should use the default standard facets listed here: https://docs.datadoghq.com/standard-attributes/?product=log. 

The DNS ones for example, should be mapped to the existing standard attributes with an attribute remapper